packages/cli/src/ceramic-daemon.ts

```diff
@@ -524,13 +528,33 @@ export class CeramicDaemon {
    * @dev Useful when the streamId is unknown, but you have the init contents
    */
   async createStreamFromGenesis(req: Request, res: Response): Promise<void> {
+    const span = Metrics.startSpan(METRIC_CREATE_STREAM_SPAN)
+    const start = new Date()
+    let closedAt
+
+    const recordClosedAt = () => {
+      const now = new Date()
+      closedAt = now.getTime() - start.getTime()
+    }
+    res.on('close', recordClosedAt)
+
     const { type, genesis, opts } = req.body
     const stream = await this.ceramic.createStreamFromGenesis(
       type,
       StreamUtils.deserializeCommit(genesis),
       opts
     )
     res.json({ streamId: stream.id.toString(), state: StreamUtils.serializeState(stream.state) })
+
+    res.removeListener('close', recordClosedAt)
+    const now = new Date()
+    const completedAt = now.getTime() - start.getTime()
```

Comment:
semantically, i would say createDuration instead of completedAt here

Reply:
I think it would be nice to have this metric record how long a create stream call takes. What's the best way to visualize this on Grafana @gvelez17 @stephhuynh18